src/Export/Adapters/SalesFrequencyAdapter.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Export\Adapters;
+
+use DateTimeImmutable;
+use FINDOLOGIC\Export\Data\SalesFrequency;
+use Shopware\Core\Content\Product\ProductEntity;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityRepository;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\EqualsFilter;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\MultiFilter;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\RangeFilter;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+
+class SalesFrequencyAdapter
+{
+    /** @var EntityRepository $orderLineItemRepository */
+    private $orderLineItemRepository;
+
+    /** @var SalesChannelContext $salesChannelContext */
+    private $salesChannelContext;
+
+    public function __construct(
+        EntityRepository $orderLineItemRepository,
+        SalesChannelContext $salesChannelContext
+    ) {
+        $this->orderLineItemRepository = $orderLineItemRepository;
+        $this->salesChannelContext = $salesChannelContext;
+    }
+
+    public function adapt(ProductEntity $product): ?SalesFrequency
```

Comment:
salesfrequency shouldn't be nullable. Either it hasn't sold, which would be `0`, or it has been sold, which would be the count of products sold.